lib/engine/game/g_18_dixie/trains.rb

```diff
@@ -73,20 +73,21 @@ module Trains
             price: 600,
             num: 2,
             variants: [
-
               name: '4D',
               distance: [{ 'nodes' => %w[city offboard town], 'pay' => 4, 'visit' => 4, 'multiplier' => 2 }],
-              events: [], # SCL Merger chance
+              events: [{ 'type' => 'scl_formation_chance' }],
               price: 800,
             ],
+            variant_event_handling: :individual,
           },
           {
             name: '5D',
             available_on: '7',
             distance: [{ 'nodes' => %w[city offboard town], 'pay' => 5, 'visit' => 5, 'multiplier' => 2 }],
-            events: [], # ICG Merger Chance
+            events: [{ 'type' => 'icg_formation_chance' }],
             price: 900,
             num: 5,
+            variant_event_handling: :individual,
```

Comment:
If I'm understanding the changes in `lib/engine/train.rb` correctly, this value prevents other variants from having the same events as this one; I see how this makes sense for the 4D, since it's a variant of the 2D, but I don't get why the 5D needs this.